tests/testthat/_snaps/demo/demo.md

```diff
@@ -315,6 +315,7 @@ cat(news, sep = "\n")
 ## # tea 0.0.0.9001
 ## 
 ## - Add tests for cup of tea.
+## 
```

Comment:
I find NEWS more concise if there is no empty line inbetween. Are there semantic differences?